### PR TITLE
Sync Travis file with the `main` branch

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Updates for missing PR preview builds

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,9 +27,9 @@ jobs:
         - pip3 install pyyaml
         - pip3 install aura.tar.gz
       script:
-        - python3 build.py --distro openshift-enterprise --product "OpenShift Container Platform" --version 4.11 --no-upstream-fetch && python3 makeBuild.py
+        - python3 build.py --distro openshift-enterprise --product "OpenShift Container Platform" --version 4.13 --no-upstream-fetch && python3 makeBuild.py
```
Comment (gaurav-nelson) [inline .travis.yml:30]: Should we change it to 4.14 in the validate step for 4.14 branch?
@aireilly @kalexand-rh WDYT?

Comment (aireilly) [reply .travis.yml:30]: Makes sense to me!

Comment (kalexand-rh) [reply .travis.yml:30]: I agree.

Comment (bergerhoffer) [reply .travis.yml:30]: Shouldn't this change be made in the main branch too though? So they stay in sync?

**Pull request merged**